### ISSUE TITLE
"Baptism"

### ISSUE THREAD

**Issue by mgoelzer** [libp2p/glasgow2018#2]
Initialize newer folks like @raulk @bigs @jhiesey by hearing from older/initial folks @jbenet @lgierth @Stebalien (eg, review the whiteboard)

**Comment by raulk** [comment]
Personally I'd like to share in the _grand vision_ of libp2p so I can (a) better serve the project, (b) take decisions in alignment with the ambitions, (c) communicate outwards with purpose and passion.

**Comment by bigs** [comment]
sounds good. one of the things mentioned previously was that libp2p might have aims to be a *general purpose* networking library, which surprised me a bit. would love to hear a bit more expansion on that, as well.

**Comment by mgoelzer** [comment]
Yep, this is exactly the kind of big picture stuff I want us to talk about.  

One way to think about it:  what TCP/IP has been for the past 50 years, we want libp2p to be for the next 50.  Currently, TCP/IP is the foundation for essentially every type of networking use case:  websites, smartphone apps, corporate networks, etc.  TCP/IP has certain properties that make it very very suitable for client-server computing, and this has contributed to the client-server world that we have today (economics of centralizing hardware is another big reason).

In the forthcoming decentralized world, libp2p could do the same thing.  It also has certain properties that lend themselves to dapps and decentralization.  The proliferation of powerful computers in everyone's pockets could change the economic equation.  That, combined with social forces (people upset over loss of privacy, personal data brokering, etc) could lead to an ecosystem of dapps that give users true control and ownership over their data.
